spec/lib/hyrax/hirmeos/metrics_tracker_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+require 'hyrax/hirmeos/metrics_tracker'
+require 'rails_helper'
+
+RSpec.describe Hyrax::Hirmeos::MetricsTracker do
+  subject(:tracker) { described_class.new }
+
+  before do
+    tracker.username = "UsernameTest"
+    tracker.password = "Password"
+    tracker.metrics_base_url = "https://metrics-api.operas-eu.org"
+    tracker.token_base_url = "https://tokens.ubiquity.press/"
+    tracker.translation_base_url = "https://translator.ubiquity.press"
```

Comment:
These should be on the class instead of the instance.  It is probably best to do this in an `around` then to make sure you reset the previous state after each test.  Or I've seen it done where these could be set in the `rails_helper` and assumed for the whole test suite.
Or just don't mess with it for now since it looks like I didn't do anything better here: https://github.com/samvera-labs/hyrax-doi/blob/main/spec/services/hyrax/doi/datacite_registrar_spec.rb#L39-L44